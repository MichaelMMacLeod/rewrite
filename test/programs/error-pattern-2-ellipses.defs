(def (this ($pattern .. has serious $problems .. (I think)))
  constructor)