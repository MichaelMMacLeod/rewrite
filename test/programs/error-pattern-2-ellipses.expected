error[E006]: too many ellipses in single term of pattern
./test/programs/error-pattern-2-ellipses.defs:1:12
   |
 1 | (def (this ($pattern .. has serious $problems .. (I think)))
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this term has more than one ellipsis
./test/programs/error-pattern-2-ellipses.defs:1:47
   |
 1 | (def (this ($pattern .. has serious $problems .. (I think)))
   |                                               ^^ ellipsis #2
help: each term in a definitions's pattern term may have at most one ellipsis
